CHANGELOG.md

```diff
@@ -349,7 +349,7 @@ and this project adheres to
   - [clang_parser] pass BPFtrace as arg instead of StructMap (a0af75f) by Matheus Marchini &lt;mat@mmarchini.me&gt;
   - [ast] introduce Identifier type to AST (389d55f) by Matheus Marchini &lt;mat@mmarchini.me&gt;
   - use CMAKE_SYSTEM_PROCESSOR when selecting whether to include x86_64 or aarch64 sources (0ea7a63) by Michał Gregorczyk &lt;michalgr@fb.com&gt;
-  - Clearify error message for mismatched llvm. (9b77fee) by George Slavin &lt;george.r.slavin@gmail.com&gt;
```

Comment:
Currently our "changelog" is just a bunch of grouped commit subject lines so I think you should leave it as is. These are typos in the commits.

Reply:
Makes sense (was unsure about that one), fixed in new version